llmfoundry/main.py

```diff
@@ -101,7 +101,7 @@ def main(cfg):
         f'torch.distributed.*_base is a private function and will be deprecated.*'
     )
 
-    cfg.dist_timeout = cfg.get('dist_timeout', 1800.0)
+    cfg.dist_timeout = cfg.get('dist_timeout', 600.0)
```

Comment:
nit: why is this a float?

Reply:
I think Composer `dist.initialize_dist` takes a float.